build.gradle

```diff
@@ -54,9 +49,9 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:$build_gradle_version"
+        classpath 'com.android.tools.build:gradle:3.2.1'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath "com.dicedmelon.gradle:jacoco-android:$jacoco_android_plugin_version"
+        classpath 'com.dicedmelon.gradle:jacoco-android:0.1.2'
```

Comment:
why do we remove version definition for those? ☝️

Reply:
Just reverting the changes from #41